src/components/Navigation/components/MobileNavigation.tsx

```diff
@@ -0,0 +1,84 @@
+import { NavigationItem } from "@/types/fsxa-ui";
+import Component from "vue-class-component";
+import BaseNavigation from "./BaseNavigation";
+
+@Component({
+  name: "MobileNavigation",
+})
+class MobileNavigation extends BaseNavigation {
+  collapsedItems: NavigationItem["key"][] = [];
+
+  render() {
+    return (
+      <div class="w-full max-h-full border-t border-b border-gray-300 flex items-stretch justify-start font-sans flex-col overflow-y-auto">
+        {this.items.map(item => (
+          <div class="last:border-b-0 relative border-b border-gray-300 bg-white">
+            <a
+              href="#"
+              onClick={event => {
+                event.preventDefault();
+                this.triggerItemClick(item);
+              }}
+              class="block px-5 py-3 tracking-xl hover:bg-gray-200 transition-colors duration-200 relative uppercase font-semibold text-gray-600 text-xs"
+            >
+              <span class="flex-grow leading-3">{item.label}</span>
+              {item.children.length ? (
+                <span
+                  class="absolute top-1/2 right-0 -mt-4 mr-3 p-1 hover:bg-white hover:shadow rounded-full"
+                  onClick={event => {
+                    event.preventDefault();
+                    event.stopImmediatePropagation();
+                    const currentIndex = this.collapsedItems.indexOf(item.key);
+                    if (currentIndex === -1) {
+                      this.collapsedItems.push(item.key);
+                    } else {
+                      const nextCollapsedItems = [...this.collapsedItems];
+                      nextCollapsedItems.splice(currentIndex, 1);
+                      this.collapsedItems = nextCollapsedItems;
+                    }
+                  }}
+                >
+                  <svg
```

Comment:
To improve readability I would put this into a function. Because from just reading the code I don't know what svg will be rendered. Is it a logo? Is it an arrow?

Reply:
I've added a reference to the icon-set.